src/core/components/hv-navigator/index.tsx

```diff
@@ -10,9 +10,9 @@ import * as NavigationContext from 'hyperview/src/contexts/navigation';
 import * as NavigatorMapContext from 'hyperview/src/contexts/navigator-map';
 import * as NavigatorService from 'hyperview/src/services/navigator';
 import * as Types from './types';
-import * as TypesLegacy from 'hyperview/src/types-legacy';
+import * as TypesLegacy from 'hyperview/src/types';
```

Comment:
We'll want to remove the 'legacy' terminology after this merge. That was in place as a stopgap while developing new functionality in TS while the bulk was in JS.